components/bl-animatedBox-component/README.md

```diff
@@ -0,0 +1,72 @@
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="780"/>
+</p>
+
+## Properties
+
+| Property                    | Type                                                                                                                                                                                                                                           | Default value | Logic | Data Binding | UI Setting | Description                                                                          |
+|-----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|-------|--------------|------------|--------------------------------------------------------------------------------------|
+| Animation Type <br> `type`  | *Select* <br> [`bounce`, `bounceIn`, `bounceOut`, `flash`, `flip`, `flipInX`, `flipInY`, `heartBeat`, `hinge`, `jackInTheBox`, `jello`, `pulse`, `rotateIn`, `rubberBand`, `shakeX`, `shakeY`, `swing`, `tada`, `wobble`, `zoomIn`, `zoomOut`] | `bounce`      |       | NO           | YES        | Controls the animation type of the component.                                        |
+| Duration <br> `duration`    | *Number*                                                                                                                                                                                                                                       | 1             |       | NO           | YES        | Controls the duration of the animation, in seconds.                                  |
+| Delay <br> `delay`          | *Number*                                                                                                                                                                                                                                       | 0             |       | NO           | YES        | Controls the duration to delay the animation, in seconds.                            |
+| Loop <br> `loop`            | *Number*                                                                                                                                                                                                                                       | 0             |       | NO           | YES        | Controls the number of times the animation should loop, 0 is equivalent to infinite. |
+| Start Play <br> `autoStart` | *Checkbox*                                                                                                                                                                                                                                     | `true`        |       | NO           | YES        | Controls whether to play animation on startup or not.                                |
```

Comment:
Could we add a property 'Play on hover' that will add a 'hovered' class and play the animation for mouse/pointer in/out events?